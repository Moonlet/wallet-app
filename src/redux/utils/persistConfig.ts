import AsyncStorage from '@react-native-community/async-storage';
import { createMigrate } from 'redux-persist';
import { migrations } from './migrations';

export const persistConfig = {
    key: 'root',
    version: 11,
    storage: AsyncStorage,
    blacklist: ['ui'],
    migrate: createMigrate(migrations, { debug: false })
};
