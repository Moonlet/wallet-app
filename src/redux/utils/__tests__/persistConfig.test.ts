import { persistConfig } from '../persistConfig';

test('should set the config', () => {
    expect(persistConfig).toEqual({ key: 'root', storage: undefined, version: 1 });
});
