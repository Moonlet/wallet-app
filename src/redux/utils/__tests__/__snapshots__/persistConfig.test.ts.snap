// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should set the config 1`] = `
Object {
  "blacklist": Array [
    "ui",
  ],
  "key": "root",
  "migrate": [Function],
  "storage": undefined,
  "version": 2,
}
`;
